src/test/java/FileUploader.java

```diff
@@ -0,0 +1,23 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.Test;
+
+import static org.testng.AssertJUnit.assertEquals;
+
+public class FileUploader extends BasePageTest {
+    public static final String URL_FILE_UPLOADER = "http://the-internet.herokuapp.com/upload";
+
+
+    @Test
+    public void fileUploader () {
+        driver.get(URL_FILE_UPLOADER);
+//Я не понимаю в чем прикол, но тест работает только с абсолютным путем
+        driver.findElement(By.id("file-upload")).sendKeys("C:\\Users\\Zagvo\\OneDrive\\Документы\\GitHub\\TMS QA05\\ClassWork\\HerokuApp\\src\\test\\resources\\Изображение_верблюда.jpg");
```

Comment:
https://stackoverflow.com/questions/45612089/how-to-upload-file-with-relative-path-in-selenium-webdriver